README.md

```diff
@@ -214,30 +380,11 @@ There are two use cases for deep linking that can be relevant here:
 
 In order for deep linking to work, there are a few configurations that are needed and these are no different from what are required for handling deep linking in general and [Apple documentation](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on this can be followed in conjunction with the steps highlighted here:
 
-### Option 1: Modify Open Tracking
-
-If you plan to use universal links in your app for deep linking you will need to modify the push open tracking as described below:
-
-```swift
-extension AppDelegate: UNUserNotificationCenterDelegate {
-    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        let handled = KlaviyoSDK().handle(notificationResponse: response, withCompletionHandler: completionHandler) { url in
-            print("deep link is ", url)
-        }
-        if !handled {
-           // not a klaviyo notification should be handled by other app code
-        }
-    }
-}
-```
-
-Note that the deep link handler will be called back on the main thread. If you want to handle uri schemes in addition to universal links you implement them as described below.
-
-### Option 2: Use URL Schemes
+##### Option 1: URI Schemes
```

Comment:
It looks like we use URL scheme everywhere else. Should it be URI or URL?

Reply:
Typically I've seen them called URI schemes but the name is interchangeable I guess. Agree maybe consistency would be clearest though.

Reply:
Going to switch to using URL since that's what Apple seems to use in their [docs](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app).

Reply:
Done.